packages/wasm-dpp/src/errors/protocol_error.rs

```diff
@@ -1,12 +1,12 @@
-use wasm_bindgen::JsValue;
+use wasm_bindgen::{JsError, JsValue};
 
 use super::consensus_error::from_consensus_error;
 
-pub fn from_protocol_error(e: dpp::ProtocolError) -> JsValue {
-    match e {
+pub fn from_protocol_error(protocol_error: dpp::ProtocolError) -> JsValue {
+    match protocol_error {
         dpp::ProtocolError::AbstractConsensusError(consensus_error) => {
             from_consensus_error(*consensus_error)
         }
-        _ => unimplemented!(),
+        e => JsError::new(&format!("Protocol error: {e}")).into(),
```

Comment:
All good, except I would bring back the `unimplemented` - it's here specifically so people who work on the code had an incentive to implement an actual underlying error